### PR TITLE
LPD-15335 Implement Functional Test to Validate Deletion of Object Relationships From Different Folders

### PR DESCRIPTION
**Pull request by rebsilva**
Related Issue: https://liferay.atlassian.net/browse/LPD-15335

### CODE REVIEW

**Comment by rebsilva** [comment]
this test was made by @igor-franca :cherry_blossom:

File: modules/test/playwright/pages/object-web/ModelBuilderPage.ts [hunk 1]
```diff
@@ -41,10 +54,33 @@ export class ModelBuilderPage {
...
+
+	async clickDeleteObjectRelationshipButton() {
+		this.deleteObjectRelationshipButton.click();
+	}
+
 	async clickFitViewButton() {
 		this.fitViewButton.click({force: true});
 	}
 
+	async clickObjectRelationshipEdge(objectRelationshipLabel: string) {
```
Comment (cgmonte) [inline modules/test/playwright/pages/object-web/ModelBuilderPage.ts:78]: Create a new method only if:
1 - It's called outside the POM
2 - It's a complex call and used more than once inside the POM.

The body of this function should be moved to `deleteObjectRelationship`.

**Review by cgmonte — CHANGES_REQUESTED** [review]
Just one small change request, @igor-franca!

**Comment by cgmonte** [comment]
ci:test:playwright

**Comment by rebsilva** [comment]
hey @igor-franca, will you do the changes or do you need my help on this? please let me know!

File: modules/test/playwright/pages/object-web/ModelBuilderPage.ts (hunk 1 above)
Comment (igor-franca) [reply modules/test/playwright/pages/object-web/ModelBuilderPage.ts:78]: Hey @cgmonte, considering the reuse potential of the `clickObjectRelationshipEdge` function. Don't you think we could call the function outside the POM right before the `deleteObjectRelationship` function?

So:

```
	await modelBuilderPage.clickObjectRelationshipEdge(
		objectRelationshipLabel
	);

	await modelBuilderPage.deleteObjectRelationship(
		objectRelationshipLabel,
		objectRelationshipName
	);
```

Comment (cgmonte) [reply modules/test/playwright/pages/object-web/ModelBuilderPage.ts:78]: Yes, I agree. We should keep this method and call it from outside.

**Comment by igor-franca** [comment]
> hey @igor-franca, will you do the changes or do you need my help on this? please let me know!

Hey @rebsilva, I'm opening another [PR](https://github.com/liferay-objects/liferay-portal/pull/3110) making the proposed changes. Thanks for taking care of this PR while I was away 😄

**Pull request closed without merging**